config/services_repositories.yml

```diff
@@ -95,6 +95,10 @@ services:
     demosplan\DemosPlanCoreBundle\Repository\SlugRepository:
         $entityClass: 'demosplan\DemosPlanCoreBundle\Entity\Slug'
 
+    demosplan\DemosPlanCoreBundle\Repository\SupportContactRepository:
+        arguments:
```

Comment:
why is this different to any other definition?

Reply:
I tried to make it work with doctrine arguments like #[ORM\Entity(repositoryClass:....] but couldnt make it work.
This was a leftover of things I tried out.... adjusted the pattern in 0e3e5065a79d5373e7849e282fbe19b39488850d